examples/pyTGCyto.ipynb

```diff
@@ -0,0 +1,144 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyTigerGraph as tg\n",
+    "import pyTGCyto\n",
+    "import ipycytoscape\n",
+    "\n",
+    "conn = tg.TigerGraphConnection(host=\"http://127.0.0.1\", restppPort=\"26900\", gsPort=\"26240\", graphname=\"MyGraph\", username=\"tigergraph\", password=\"tigergraph\", apiToken=\"el1op7a9eqrlq4ape5t452lukv991k7h\")\n",
```

Comment:
We should not be committing api tokens into the repo. Better get them from the user using getpass https://docs.python.org/2/library/getpass.html